.github/workflows/zips.yml

```diff
@@ -113,59 +116,98 @@ jobs:
           name: zip-files
           path: ${{ github.ref }}
 
-  deploy-to-wiki:
-    name: Deploy To Wiki
+  upload-to-cloud-storage:
+    name: Upload to GCS
     runs-on: ubuntu-latest
     needs: build-zips
     steps:
-      - uses: actions/checkout@v3
-        with:
-          repository: ${{ github.repository }}.wiki
-          # GitHub Wikis use this term as their main branch; this cannot be
-          # changed so we need to continue using it for now.
-          ref: master
-      - name: Download artifacts
-        uses: actions/download-artifact@v3
+      - name: Set branch environment variables
+        id: branch-target
+        if: github.event_name != 'pull_request'
+        run: echo "dir=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
+      - name: Set PR environment variables
+        id: pr-target
+        if: github.event_name == 'pull_request'
+        run: echo "dir=pull/${{ github.event.number }}" >> $GITHUB_OUTPUT
+      - uses: actions/download-artifact@v3
         with:
           name: zip-files
           path: ${{ github.ref }}
-      - name: Commit updates
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+      - uses: google-github-actions/setup-gcloud@v1
+      - name: Upload PR files
+        run: gcloud storage cp --recursive ${{ github.ref }} gs://${{ env.BUCKET_NAME }}/builds/${TARGET_DIR}
+        env:
+          TARGET_DIR: ${{ steps.branch-target.outputs.dir }}${{ steps.pr-target.outputs.dir }}
+      - name: Check the comment exists
```

Comment:
Let's keep this as a separate job as it will only be needed for PRs. This will take a small amount of extra time, but the job [only takes a few seconds](https://github.com/google/site-kit-wp/actions/runs/4721604126/jobs/8375349284).

Reply:
Ok, updated.